01week/resume/index.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
```

Comment:
Little hard to read since its not indented correctly.
I am not sure if that is because you are working on a windows machine.